modules/rosa-classic-aws-rhcos-ami.adoc

```diff
@@ -0,0 +1,969 @@
+|`rhcos-411.85.202203242008-0-x86_64`
+|`ami-00c6ece95bdb1f008`
+|`rhcos-410.84.202111201002-0-x86_64`
+|`ami-03d94dd5fe4f2c7e4`
+|`rhcos-48.84.202105062123-0`
+|`ami-049899b3091f4732c`
+|`rhcos-46.82.202008181646-0`
+|`ami-013fba3192d95f465`
+|`rhcos-46.82.202008071940-0`
+|`ami-0d8e30e0eba1db1ae`
+|`rhcos-413.86.202301302144-0-x86_64`
+|`ami-0d470a464b7276f8e`
+|`rhcos-416.94.202403071059-0-x86_64`
+|`ami-0ec5f732a717b9d7e`
+|`rhcos-414.92.202305090606-0-x86_64`
+|`ami-0f48f7443cfdde283`
+|`rhcos-43.82.202008010953.0-hvm`
+|`ami-079a33da05df563be`	
+|`[Copied ami-05f09360cb1869045 from us-east-1] rhcos-44.81.202007071934-0-hvm`
+|`ami-0febcb2d991a09af6`
+|`rhcos-42.80.20191002.0-hvm`
+|`ami-000fd95e97985af95`
+|`rhcos-415.92.202309201557-0-x86_64`
+|`ami-0b16f6379bd822e53`
+|`rhcos-44.82.202008011133-0-hvm`
+|`ami-090e50abafe798e40`
+|`rhcos-49.84.202107191624-0`
+|`ami-04d23b757c7910bf5`
+|`rhcos-48.84.202107202156-0`
+|`ami-093fb5d8f5e210e74`
+|`rhcos-45.82.202008010929-0`
+
+.19+|*ca-central-1*
+|`ami-0f5bea0dbf22330ce`
+|`rhcos-44.82.202008011133-0-hvm`
+|`ami-045ac5f61390f225d`
+|`rhcos-45.82.202008010929-0`
+|`ami-0e30703612613eb7f`
+|`rhcos-48.84.202105062123-0`
+|`ami-026fee6fd26d2e93e`
+|`rhcos-46.82.202008071940-0`
+|`ami-0ae0e48e9fe4548d8`
+|`rhcos-43.82.202008010953.0-hvm`
+|`ami-007f3afa4cb5c8f3b`
+|`rhcos-47.83.202102090044-0`
+|`ami-03e3d037fa56ba391`
+|`rhcos-414.92.202305090606-0-x86_64`
+|`ami-0686a827d05ff9d10`
+|`rhcos-46.82.202008181646-0`
+|`ami-00b56d4237359f57e`
+|`rhcos-410.84.202111201002-0-x86_64`
+|`ami-0ac625dd75f4875b8`
+|`rhcos-416.94.202403071059-0-x86_64`
+|`ami-0a668d27ee6f2d05b`
+|`rhcos-415.92.202309201557-0-x86_64`
+|`ami-0708cfc5575d806d7`
+|`rhcos-413.86.202301302144-0-x86_64`
+|`ami-0e477c70da6751d1e`
+|`[Copied ami-05f09360cb1869045 from us-east-1] rhcos-44.81.202007071934-0-hvm`
+|`ami-0fedf5b459c5eee36`
+|`rhcos-45.81.202005200134-0`
+|`ami-0480eaa4de0b83565`
+|`rhcos-49.84.202107191624-0`
+|`ami-03ecefb6ad113a678`
+|`rhcos-48.84.202107202156-0`
+|`ami-0e8480cffce65c74d`
+|`rhcos-411.85.202203242008-0-x86_64`
+|`ami-049d5b469f54f8fb8`
+|`rhcos-42.80.20191002.0-hvm`
+|`ami-042379067546d8531`
+|`rhcos-412.86.202207142104-0-x86_64`
+
+.19+|*eu-central-1*
+|`ami-07bd2390ee0c2b2eb`
+|`rhcos-47.83.202102090044-0`
+|`ami-043bb25ed4a6b3871`
+|`rhcos-45.81.202005200134-0`
+|`ami-0baa8e69798198b4f`
+|`rhcos-48.84.202107202156-0`
+|`ami-07dc1cd741b71b502`
+|`rhcos-43.82.202008010953.0-hvm`
+|`ami-072a4b61d61802fc5`
+|`rhcos-415.92.202309201557-0-x86_64`
+|`ami-0da607360203ff018`
+|`rhcos-413.86.202301302144-0-x86_64`
+|`ami-05714bcec6aa9a156`
+|`rhcos-49.84.202107191624-0`
+|`ami-06b9c321d58aac43e`
+|`rhcos-42.80.20191002.0-hvm`
+|`ami-0472294cbab427817`
+|`rhcos-46.82.202008071940-0`
+|`ami-06ee906c34a6bc206`
+|`rhcos-48.84.202105062123-0`
+|`ami-0459011ee17af501c`
+|`rhcos-411.85.202203242008-0-x86_64`
+|`ami-04c056b5a7e86668b`	
+|`[Copied ami-05f09360cb1869045 from us-east-1] rhcos-44.81.202007071934-0-hvm`
```

Comment:
Can we clarify why the `[Copied ami-.... from us-east-1]` is there? Can we remove it?